src/pages/index.js

```diff
@@ -96,54 +123,136 @@ function closeModal(modal) {
   document.removeEventListener("keydown", handleEscClose);
 }
 
+function disableButton(evt) {
+  if (cardNameInput.value != "" && cardLinkInput.value != "") {
+    cardSubmitBtn.disabled = false;
+    cardSubmitBtn.classList.remove("modal__submit-btn_disabled");
+  } else {
+    cardSubmitBtn.classList.add("modal__submit-btn_disabled");
+    cardSubmitBtn.disabled = true;
```

Comment:
Please, delete it

You should reuse the `disableButton` function from the `validation.js` file. You should **export** it from there